src/i965_encoder_vp8.c

```diff
@@ -0,0 +1,6192 @@
+    0x4b3d3a38,
+    0x4b3d3a38,
+    0x4b3d3a38,
+    0x4b3d3a38,
+    0x4b3d3a38,
+    0x4c3e3b38,
+    0x4c3e3b38,
+    0x4c3e3b38,
+    0x4c3e3b38,
+    0x4d483c39,
+    0x4d483c39,
+    0x4d483c39,
+    0x4d483c39,
+    0x4d483c39,
+    0x4e483e3a,
+    0x4e483e3a,
+    0x4e483e3a,
+    0x4e483e3a,
+    0x4f493f3b,
+    0x4f493f3b,
+    0x4f493f3b,
+    0x4f493f3b,
+    0x584a483c,
+    0x584a483c,
+    0x584a483c,
+    0x584a483c,
+    0x594b483d,
+    0x594b483d,
+    0x594b483d,
+    0x594b483d,
+    0x594b493e,
+    0x594b493e,
+    0x594b493e,
+    0x594b493e,
+    0x5a4c493f,
+    0x5a4c493f,
+    0x5a4c493f,
+    0x5a4c493f,
+    0x5b4d4a48,
+    0x5b4d4a48,
+    0x5b4d4a48,
+    0x5b4d4a48,
+    0x5b4d4b48,
+    0x5b4d4b48,
+    0x5b4d4b48,
+    0x5b4d4b48,
+    0x5b4d4b48,
+    0x5c4e4b48,
+    0x5c4e4b48,
+    0x5c4e4b48,
+    0x5c4e4b48,
+    0x5c4f4c49,
+    0x5c4f4c49,
+    0x5c4f4c49,
+    0x5c4f4c49,
+    0x5d584c49,
+    0x5d584c49,
+    0x5d584c49,
+    0x5e584d4a,
+    0x5e584d4a,
+    0x5e584e4a,
+    0x5e584e4a,
+    0x5f594e4b,
+    0x5f594e4b,
+    0x5f594e4b,
+    0x5f594f4b,
+    0x5f594f4b,
+    0x68594f4c,
+    0x68594f4c,
+    0x685a584c,
+    0x685a584c,
+    0x685a584c,
+    0x685a584c,
+    0x695b584d,
+    0x695b594d,
+    0x695b594d,
+    0x695b594e,
+    0x695b594e,
+    0x6a5c594e,
+    0x6a5c594e,
+    0x6a5c594f,
+    0x6a5c594f,
+    0x6a5c5a4f,
+    0x6a5c5a4f,
+    0x6a5d5a58,
+    0x6b5d5a58,
+    0x6b5d5a58,
+    0x6b5d5b58,
+    0x6b5e5b58,
+};
+
+const unsigned short
+brc_quant_dc_vp8[128] = {
+    4,   5,   6,   7,   8,   9,  10,  10,  11,  12,  13,  14,  15,  16,  17,  17,
+    18,  19,  20,  20,  21,  21,  22,  22,  23,  23,  24,  25,  25,  26,  27,  28,
+    29,  30,  31,  32,  33,  34,  35,  36,  37,  37,  38,  39,  40,  41,  42,  43,
+    44,  45,  46,  46,  47,  48,  49,  50,  51,  52,  53,  54,  55,  56,  57,  58,
+    59,  60,  61,  62,  63,  64,  65,  66,  67,  68,  69,  70,  71,  72,  73,  74,
+    75,  76,  76,  77,  78,  79,  80,  81,  82,  83,  84,  85,  86,  87,  88,  89,
+    91,  93,  95,  96,  98, 100, 101, 102, 104, 106, 108, 110, 112, 114, 116, 118,
+    122, 124, 126, 128, 130, 132, 134, 136, 138, 140, 143, 145, 148, 151, 154, 157
+};
+
+const unsigned short
+brc_quant_ac_vp8[128] = {
+    4,   5,   6,   7,   8,   9,  10,  11,  12,  13,  14,  15,  16,  17,  18,  19,
+    20,  21,  22,  23,  24,  25,  26,  27,  28,  29,  30,  31,  32,  33,  34,  35,
+    36,  37,  38,  39,  40,  41,  42,  43,  44,  45,  46,  47,  48,  49,  50,  51,
+    52,  53,  54,  55,  56,  57,  58,  60,  62,  64,  66,  68,  70,  72,  74,  76,
+    78,  80,  82,  84,  86,  88,  90,  92,  94,  96,  98, 100, 102, 104, 106, 108,
+    110, 112, 114, 116, 119, 122, 125, 128, 131, 134, 137, 140, 143, 146, 149, 152,
+    155, 158, 161, 164, 167, 170, 173, 177, 181, 185, 189, 193, 197, 201, 205, 209,
+    213, 217, 221, 225, 229, 234, 239, 245, 249, 254, 259, 264, 269, 274, 279, 284
+};
+
+const unsigned short
```

Comment:
The definition of const table is no problem. 
Can we add the static prefix for the const table so that it is explicitly defined as local const?